swift/vectors.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+
+struct Vector: CustomStringConvertible {
+    
+    var x: Double
+    var y: Double
+    
+    init(x: Double, y: Double){
+        self.x = x;
+        self.y = y;
+    }
+    
+    func magnititude() -> Double {
+        return pow(x*x + y*y, 1/2)
+    }
+    
+    var description: String {
+        return "<\(x),\(y)>"
+    }
+}
+
+func + (left: Vector, right: Vector) -> Vector {
+    return Vector(x: left.x + right.x, y: left.y + right.y)
+}
+
+func * (left: Vector, right: Vector) -> Double {
+    return left.x * right.x + left.y * right.y
+}
+
+let u = Vector(x: 3,y: 4)
+let v = Vector(x: -5,y: 10)
```

Comment:
lol its okay. I'm bored at JFK right now so it's keeping me entertained. just threw up some fixed stuff